tasks/main.yml

```diff
@@ -52,10 +56,12 @@
     interface_responses: >-
       {{ {dell_powerconnect_interface_prompt:
           (['description ' ~ item.value.description] if 'description' in item.value else []) +
+          (['name ' ~ item.value.name] if 'name' in item.value else []) +
```

Comment:
Any reason not to just include this as a config line?

Reply:
Hi @markgoddard no, just purely cosmetic. Idea was to have the same layout than with 'description' which works as a 'header' for the next block (config); I'm reading my own commit message and it sounds a bit misleading indeed..

Reply:
Does the name argument apply to non-VLAN interfaces? Just a bit worried it's quite specific.

If we do add it, we'll need to update the README and comments in defaults/main.yml.

Reply:
I'm not sure I understand the question, but I'd say no:
```
myswitch(config)# int gi1/0/99
myswitch(config-if)# name xxx
% Unrecognized command
```
So, yes, it seems a bit specific, at least for the 55xx. Another configuration option is to use the global config with "vlan database; name xxx", but you can't set the "ip address" in there.

I still personally prefer listing it with 'name', so I'm providing a patch to the files you mention, but I understand your remarks... out of curiosity, how have you been configuring vlans then? With name as a 'config'?

Reply:
Normally we configure the name using `vlan database; name xxx` in global config, as you suggest. Equally you could put it on the VLAN interface, although isn't that more about the L3 interface for the VLAN than the name of the VLAN itself? Perhaps there is no distinction.